modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/schema/DataDefinition.es.js

```diff
@@ -28,4 +30,12 @@ export class DataDefinitionSchema extends Schema {
 	get contentType() {
 		return this[SYMBOL_RAW].dataDefinition.contentType;
 	}
+
+	get defaultDataLayout() {
+		return new DataLayoutSchema(this[SYMBOL_RAW]);
+	}
```

Comment:
I ended up removing this property from the `useData` side, do you have any use cases that use this? apparently, he is the same that DataLayout but I still don't understand the need for him.